samples/dotnet/EventHub/KafkaOutputManyWithHeaders.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Extensions.Kafka;
+using Microsoft.Extensions.Logging;
+
+namespace Eventhub
+{
+    public class KafkaOutputManyWithHeaders
+    {
+        [FunctionName("KafkaOutputManyWithHeaders")]
+        public static IActionResult Output(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req,
+            [Kafka("BrokerList",
+                    "topic",
+                    Username = "$ConnectionString",
+                    Password = "%EventHubConnectionString%",
```

Comment:
Please add in the documentation that password is strongly recommended to be put in the appsettings file. This will be useful to the customer to understand the risk, as we don't enforce this through validation in the extension

Reply:
Added in local settings section of Readme.